app/views/admin/accounting/transactions/_transactions_grid_definition.html.slim

```diff
@@ -1,6 +1,10 @@
 = define_grid(@transactions_grid, hide_csv_button: true,
     html: {class: 'rowlink table-hover', data: {link: 'row'}}) do |g|
 
+  - if action_name == 'show'
```

Comment:
Why is `action_name` used here? Are there cases where we would not be showing the link?

I know we sometimes export tables, but a different syntax would be used here.

Reply:
This is because of the transactions index link on the sidebar
It breaks since some of those transactions don't have loans

Reply:
Oh. I see what you mean.